src/common/Button/styles.less

```diff
@@ -3,11 +3,19 @@
 @import (reference) '~@stremio/stremio-colors/less/stremio-colors.less';
 
 .button-container {
+    // IOS specific 
+    // prevents showing the default context-menu when long pressing an anchor in safari.
+    -webkit-touch-callout: none !important;
+    // prevents user from selecting text from a div on long pressing in safari.
+    -webkit-user-select: none !important;
```

Comment:
`-webkit-user-select: none;` can be moved to the app's styles.
but I don't know if moving `-webkit-touch-callout` there is a good idea, as this will disable the default context menu, and I dunno if it's needed (for example in the addons section). if it's not needed anywhere I can move it to the app's styles.